tools/CMakeLists.txt

```diff
@@ -56,6 +56,12 @@ set_property(TARGET HgrToMondriaanMtx PROPERTY CXX_STANDARD_REQUIRED ON)
 add_executable(EvaluateMondriaanPartition evaluate_mondriaan_partition.cc)
 set_property(TARGET EvaluateMondriaanPartition PROPERTY CXX_STANDARD 17)
 set_property(TARGET EvaluateMondriaanPartition PROPERTY CXX_STANDARD_REQUIRED ON)
+add_executable(CalculateEpsilon calculate_epsilon.cc)
+set_property(TARGET CalculateEpsilon PROPERTY CXX_STANDARD 17)
```

Comment:
maybe rename it to CalculateRelaxedEpsilon

Reply:
Done